helpers/listDB.js

```diff
@@ -1,7 +1,9 @@
 const lists = JSON.parse(process.env.LIST_IDS);
 
 function getId(sectionName) {
-  return lists[sectionName];
+  const list = lists[sectionName];
+  if (list) return list;
```

Comment:
I have absolutely nothing against an if notation without curly brackets but you should know Next prefer it with them.